docqueries/or_tools/multiple_knapsack.result

```diff
@@ -0,0 +1,96 @@
+BEGIN;
+BEGIN
+SET client_min_messages TO NOTICE;
+SET
+DROP TABLE IF EXISTS multiple_knapsack_query CASCADE;
+NOTICE:  table "multiple_knapsack_query" does not exist, skipping
+DROP TABLE
+CREATE TABLE multiple_knapsack_query(
+  weight INTEGER,
+  cost INTEGER);
+CREATE TABLE
+INSERT INTO multiple_knapsack_query (weight,  cost)
+VALUES
+(48, 10),
+(30, 30),
+(42, 25),
+(36, 50),
+(36, 35),
+(48, 30), 
+(42, 15), 
+(42, 40),
+(36, 30),
+(24, 35), 
+(30, 45), 
+(30, 10), 
+(42, 20), 
+(36, 30), 
+(36, 25);
+INSERT 0 15
+SELECT * 
+FROM multiple_knapsack_query;
+ weight | cost 
+--------+------
+     48 |   10
+     30 |   30
+     42 |   25
+     36 |   50
+     36 |   35
+     48 |   30
+     42 |   15
+     42 |   40
+     36 |   30
+     24 |   35
+     30 |   45
+     30 |   10
+     42 |   20
+     36 |   30
+     36 |   25
+(15 rows)
+
+SELECT * 
+FROM vrp_multiple_knapsack('SELECT * FROM multiple_knapsack_query', ARRAY[100,100,100,100,100]);
+NOTICE:  Entering Mulitple Knapsack program
+NOTICE:  Starting Execution of inner query
+INFO:  ('Number of rows processed : ', 15)
+NOTICE:  SQL query returned expected column names
+NOTICE:  SQL query returned expected column types
+NOTICE:  Finished Execution of inner query
+NOTICE:  SCIP solver ready!
+INFO:  ('Total value =', 395.0)
+INFO:  ('Bin :', 0)
+INFO:  Item 1 - weight: 30 value: 30
+INFO:  Item 3 - weight: 36 value: 50
+INFO:  Item 10 - weight: 30 value: 45
+INFO:  ('Packed bin weight', 96)
+INFO:  ('Packed bin value', 125)
+INFO:  ('Bin :', 1)
+INFO:  Item 2 - weight: 42 value: 25
+INFO:  Item 14 - weight: 36 value: 25
+INFO:  ('Packed bin weight', 78)
+INFO:  ('Packed bin value', 50)
+INFO:  ('Bin :', 2)
+INFO:  Item 4 - weight: 36 value: 35
+INFO:  Item 9 - weight: 24 value: 35
+INFO:  Item 13 - weight: 36 value: 30
+INFO:  ('Packed bin weight', 96)
+INFO:  ('Packed bin value', 100)
+INFO:  ('Bin :', 3)
+INFO:  Item 8 - weight: 36 value: 30
+INFO:  Item 12 - weight: 42 value: 20
+INFO:  ('Packed bin weight', 78)
+INFO:  ('Packed bin value', 50)
+INFO:  ('Bin :', 4)
+INFO:  Item 5 - weight: 48 value: 30
+INFO:  Item 7 - weight: 42 value: 40
+INFO:  ('Packed bin weight', 90)
+INFO:  ('Packed bin value', 70)
+INFO:  ('Total packed weight', 438)
+NOTICE:  Exiting Multiple Knapsack program
+ vrp_multiple_knapsack 
+-----------------------
+ Success
```

Comment:
Same as above.

Example:
INPUT: `(id, weight, cost)` in the inner query
OUTPUT: `(bin_number, item_id)`